### PR TITLE
[improvement](exec) add more debug info on fragment exec error

### PR DESCRIPTION
**Pull request by zhannngchen**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/runtime/fragment_mgr.cpp [hunk 1]
```diff
@@ -388,6 +388,10 @@ void FragmentExecState::coordinator_callback(const Status& status, RuntimeProfil
 
     VLOG_DEBUG << "reportExecStatus params is "
                << apache::thrift::ThriftDebugString(params).c_str();
+    if (!exec_status.ok()) {
+        LOG(INFO) << "report error status: " << exec_status.to_string()
```
Comment (morningman) [inline be/src/runtime/fragment_mgr.cpp:392]:
```suggestion
        LOG(WARNING) << "report error status: " << exec_status.to_string()
```

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.23 seconds
 load time: 598 seconds
 storage size: 17154810471 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221102110816_clickbench_pr_38322.html

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**